### ISSUE TITLE
Onedrive Backup doesn't start

### ISSUE THREAD

**Issue by churchofnoise** [lavinir/hassio-onedrive-backup#84]
The add-on doesn't start anymore and as such is not ensuring back-ups are being made.
multiple attempts at starting don't work, and no indication in the log (nor supervisor, nor the Onedrive Backup addon) give any information.

Looking at the files in the backup directory, I'd say the last was made on March 30, so just before the Home Assistant 2023.4 update.

Could it be related to that?

**Comment by lavinir** [comment]
I doubt it's related.
What exactly happens when you try to start it ? Does it not go into a "running" state ? And you see no logs anywhere ?

**Comment by lavinir** [comment]
When the addon is stopped logs are cleared. Did you check the supervisor logs ?

**Comment by churchofnoise** [comment]
I did, this is all I get:
![image](https://user-images.githubusercontent.com/65717719/236669923-fe79edf2-e357-4506-856e-c09f3d4a83ea.png)

**Comment by lavinir** [comment]
What "button" resets the settings ? You uninstalled the addon and it kept the settings ? Something is not right threre. The addon does not persist anything (everything is inside it's own container). 
Each time I had re-installed I started with a clean slate. Are you running this on Hass OS ?

**Comment by churchofnoise** [comment]
I'll clarify:

- in the add-on config, when you click on the three dots on the top right, there is a possibility to reset the settings to default. Tried this, no positive effect.
- as another possibility, I uninstalled and reinstalled the add-on. same effect.
- I run Home Assistant Supervised on Debian 
Home Assistant 2023.5.2
Supervisor 2023.04.1
Frontend 20230503.3 - latest
- full disclosure: running Debian Bookworm so not officially supported by HA
